tests/plugins/image2/resizerwithreadonly.js

```diff
@@ -42,13 +42,37 @@
 					listener.removeListener();
 
 					resume( function() {
-						assert.areSame( 2, editor.editable().getElementsByTag( 'span' ).$.length );
+						assert.areSame( 3, editor.editable().getElementsByTag( 'span' ).$.length );
```

Comment:
Let's be more precise here and check if widget really contains resizer element. Also, it would be nice to specify image widget element, so we are sure that the resizer is a part of the widget. Small code example which could lead you on the correct track:

```javascript
var resizer = editor.editable().findOne( '.cke_widget_image .cke_image_resizer' );
assert.isTrue( Boolean( resizer ), 'Resizer should be enabled'  );
```

Improve also the rest of the tests in this file with recommended changes.